<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>接口自动化测试报告,测试结果如下：</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    
<style type="text/css" media="screen">
body        { font-family: verdana, arial, helvetica, sans-serif; font-size: 80%; }
table       { font-size: 100%; }
pre         { }

/* -- heading ---------------------------------------------------------------------- */
h1 {
	font-size: 16pt;
	color: gray;
}
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}

.heading .attribute {
    margin-top: 1ex;
    margin-bottom: 0;
}

.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}

/* -- css div popup ------------------------------------------------------------------------ */
a.popup_link {
}

a.popup_link:hover {
    color: red;
}

.img{
	width: 100%;
	height: 100%;
	border-collapse: collapse;
    border: 2px solid #777;
}

.screenshots {
    z-index: 100;
	position:absolute;
	left: 23%;
	top: 20%;
	display: none;
}
.close_shots {
	position:absolute;
	top:0; left:98%;
	z-index:99;
	width:20px;
}
.popup_window {
    display: none;
    position: relative;
    left: 0px;
    top: 0px;
    /*border: solid #627173 1px; */
    padding: 10px;
    background-color: #E6E6D6;
    font-family: "Lucida Console", "Courier New", Courier, monospace;
    text-align: left;
    font-size: 8pt;
    width: 800px;
}

}
/* -- report ------------------------------------------------------------------------ */
#show_detail_line {
    margin-top: 3ex;
    margin-bottom: 1ex;
}
#result_table {
    width: 80%;
    border-collapse: collapse;
    border: 1px solid #777;
}
#header_row {
    font-weight: bold;
    color: white;
    background-color: #777;
}
#result_table td {
    border: 1px solid #777;
    padding: 2px;
}
#total_row  { font-weight: bold; }
.passClass  { background-color: #6c6; }
.failClass  { background-color: #c60; }
.errorClass { background-color: #c00; }
.passCase   { color: #6c6; font-weight: bold;}
.failCase   { color: #c60; font-weight: bold; }
.errorCase  { color: #c00; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }


/* -- ending ---------------------------------------------------------------------- */
#ending {
}
.piechart{
    position:absolute;  ;
    top:20px;
    left:350px;
    width: 200px;
    float: left;
    display:  inline;
}

</style>

</head>
<body>
<script language="javascript" type="text/javascript"><!--
output_list = Array();

/* 增加显示通过分类列表*/
/* level - 0:Summary; 1:Failed; 2:Pass; 3:All */
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level == 2 || level == 0) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level < 2) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
    }
}


function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        tid0 = 't' + cid.substr(1) + '.' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        if (toHide) {
            document.getElementById('div_'+tid).style.display = 'none'
            document.getElementById(tid).className = 'hiddenRow';
        }
        else {
            document.getElementById(tid).className = '';
        }
    }
}


function showTestDetail(div_id){
    var details_div = document.getElementById(div_id)
    var displayState = details_div.style.display
    // alert(displayState)
    if (displayState != 'block' ) {
        displayState = 'block'
        details_div.style.display = 'block'
    }
    else {
        details_div.style.display = 'none'
    }
}


function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}
/* 增加饼图显示 */
function drawCircle(pass, fail, error){
    var color = ["#6c6","#c60","#c00"];
    var data = [pass,fail,error];
    var text_arr = ["pass", "fail", "error"];

    var canvas = document.getElementById("circle");
    var ctx = canvas.getContext("2d");
    var startPoint=0;
    var width = 20, height = 10;
    var posX = 112 * 2 + 20, posY = 30;
    var textX = posX + width + 5, textY = posY + 10;
    for(var i=0;i<data.length;i++){
        ctx.fillStyle = color[i];
        ctx.beginPath();
        ctx.moveTo(112,84);
        ctx.arc(112,84,84,startPoint,startPoint+Math.PI*2*(data[i]/(data[0]+data[1]+data[2])),false);
        ctx.fill();
        startPoint += Math.PI*2*(data[i]/(data[0]+data[1]+data[2]));
        ctx.fillStyle = color[i];
        ctx.fillRect(posX, posY + 20 * i, width, height);
        ctx.moveTo(posX, posY + 20 * i);
        ctx.font = 'bold 14px';
        ctx.fillStyle = color[i];
        var percent = text_arr[i] + ":"+data[i];
        ctx.fillText(percent, textX, textY + 20 * i);

    }
}

function show_shots(obj) {
	obj.nextElementSibling.style.display="block";

}

function close_shots(obj) {
	obj.parentElement.style.display="none";
}



/* obsoleted by detail in <div>
function showOutput(id, name) {
    var w = window.open("", //url
                    name,
                    "resizable,scrollbars,status,width=800,height=450");
    d = w.document;
    d.write("<pre>");
    d.write(html_escape(output_list[id]));
    d.write("\n");
    d.write("<a href='javascript:window.close()'>close</a>\n");
    d.write("</pre>\n");
    d.close();
}
*/
--></script>
<div class="piechart">
    <div>
        <canvas id="circle" width="350" height="168" ></canvas>
    </div>
</div>
<div class='heading'>
<h1>接口自动化测试报告,测试结果如下：</h1>
<p class='attribute'><strong>测试人员:</strong> Stephen</p>
<p class='attribute'><strong>开始时间:</strong> 2022-01-02 15:21:06</p>
<p class='attribute'><strong>合计耗时:</strong> 0:00:37.597547</p>
<p class='attribute'><strong>测试结果:</strong> 总共 99 通过 66 失败 27 错误 6 通过率 = 66.67%</p>

<p class='description'>用例执行情况：</p>
</div>



<p id='show_detail_line'>显示
<a href='javascript:showCase(0)'>概要</a>
<a href='javascript:showCase(1)'>失败[27]</a>
<a href='javascript:showCase(2)'>通过[66]</a>
<a href='javascript:showCase(3)'>所有[99]</a>
</p>
<table id='result_table'>
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row'>
    <td>用例集/测试用例</td>
    <td>总计</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>视图</td>
    <td>错误截图</td>
</tr>

<tr class='errorClass'>
    <td>Case.case1_案例与法规.test_案例.TestLawInfo</td>
    <td>9</td>
    <td>3</td>
    <td>0</td>
    <td>6</td>
    <td><a href="javascript:showClassDetail('c1',9)">详情</a></td>
    <td>&nbsp;</td>
</tr>

    <tr id='ft1.1' class='none'>
        <td class='errorCase'><div class='testcase'>test_CaseInfo_00001: test_CaseInfo_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.1')" >
            错误</a>

        <div id='div_ft1.1' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.1').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft1.1: case1_1
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 444, in wrap_socket
    cnx.do_handshake()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1894, in do_handshake
    self._raise_ssl_error(self._ssl, result)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1632, in _raise_ssl_error
    _raise_current_error()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\_util.py", line 57, in exception_from_error_queue
    raise exception_type(errors)
OpenSSL.SSL.Error: [('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 594, in urlopen
    self._prepare_proxy(conn)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 815, in _prepare_proxy
    conn.connect()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connection.py", line 349, in connect
    self.sock = ssl_wrap_socket(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\ssl_.py", line 359, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 450, in wrap_socket
    raise ssl.SSLError('bad handshake: %r' % e)
ssl.SSLError: ("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])",)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 439, in send
    resp = conn.urlopen(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 637, in urlopen
    retries = retries.increment(method, url, error=e, _pool=self,
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\retry.py", line 398, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case1_案例与法规\test_案例.py", line 75, in test_CaseInfo
    response = ad.do_post(url,json.dumps(payloads),headers=headers)
  File "D:\Projects\PycharmProjects\unittest\common\request.py", line 9, in do_post
    return requests.post(url=url,data=data,headers=headers,**kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 530, in request
    resp = self.send(prep, **send_kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 643, in send
    r = adapter.send(request, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft1.2' class='none'>
        <td class='errorCase'><div class='testcase'>test_CaseInfo_00002: test_CaseInfo_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.2')" >
            错误</a>

        <div id='div_ft1.2' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.2').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft1.2: case1_2
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 444, in wrap_socket
    cnx.do_handshake()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1894, in do_handshake
    self._raise_ssl_error(self._ssl, result)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1632, in _raise_ssl_error
    _raise_current_error()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\_util.py", line 57, in exception_from_error_queue
    raise exception_type(errors)
OpenSSL.SSL.Error: [('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 594, in urlopen
    self._prepare_proxy(conn)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 815, in _prepare_proxy
    conn.connect()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connection.py", line 349, in connect
    self.sock = ssl_wrap_socket(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\ssl_.py", line 359, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 450, in wrap_socket
    raise ssl.SSLError('bad handshake: %r' % e)
ssl.SSLError: ("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])",)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 439, in send
    resp = conn.urlopen(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 637, in urlopen
    retries = retries.increment(method, url, error=e, _pool=self,
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\retry.py", line 398, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case1_案例与法规\test_案例.py", line 75, in test_CaseInfo
    response = ad.do_post(url,json.dumps(payloads),headers=headers)
  File "D:\Projects\PycharmProjects\unittest\common\request.py", line 9, in do_post
    return requests.post(url=url,data=data,headers=headers,**kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 530, in request
    resp = self.send(prep, **send_kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 643, in send
    r = adapter.send(request, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft1.3' class='none'>
        <td class='errorCase'><div class='testcase'>test_CaseInfo_00003: test_CaseInfo_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.3')" >
            错误</a>

        <div id='div_ft1.3' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.3').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft1.3: case1_3
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 444, in wrap_socket
    cnx.do_handshake()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1894, in do_handshake
    self._raise_ssl_error(self._ssl, result)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1632, in _raise_ssl_error
    _raise_current_error()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\_util.py", line 57, in exception_from_error_queue
    raise exception_type(errors)
OpenSSL.SSL.Error: [('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 594, in urlopen
    self._prepare_proxy(conn)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 815, in _prepare_proxy
    conn.connect()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connection.py", line 349, in connect
    self.sock = ssl_wrap_socket(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\ssl_.py", line 359, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 450, in wrap_socket
    raise ssl.SSLError('bad handshake: %r' % e)
ssl.SSLError: ("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])",)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 439, in send
    resp = conn.urlopen(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 637, in urlopen
    retries = retries.increment(method, url, error=e, _pool=self,
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\retry.py", line 398, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case1_案例与法规\test_案例.py", line 75, in test_CaseInfo
    response = ad.do_post(url,json.dumps(payloads),headers=headers)
  File "D:\Projects\PycharmProjects\unittest\common\request.py", line 9, in do_post
    return requests.post(url=url,data=data,headers=headers,**kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 530, in request
    resp = self.send(prep, **send_kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 643, in send
    r = adapter.send(request, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft1.4' class='none'>
        <td class='errorCase'><div class='testcase'>test_CaseInfo_00004: test_CaseInfo_00004</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.4')" >
            错误</a>

        <div id='div_ft1.4' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.4').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft1.4: case1_4
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 444, in wrap_socket
    cnx.do_handshake()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1894, in do_handshake
    self._raise_ssl_error(self._ssl, result)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1632, in _raise_ssl_error
    _raise_current_error()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\_util.py", line 57, in exception_from_error_queue
    raise exception_type(errors)
OpenSSL.SSL.Error: [('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 594, in urlopen
    self._prepare_proxy(conn)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 815, in _prepare_proxy
    conn.connect()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connection.py", line 349, in connect
    self.sock = ssl_wrap_socket(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\ssl_.py", line 359, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 450, in wrap_socket
    raise ssl.SSLError('bad handshake: %r' % e)
ssl.SSLError: ("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])",)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 439, in send
    resp = conn.urlopen(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 637, in urlopen
    retries = retries.increment(method, url, error=e, _pool=self,
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\retry.py", line 398, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case1_案例与法规\test_案例.py", line 75, in test_CaseInfo
    response = ad.do_post(url,json.dumps(payloads),headers=headers)
  File "D:\Projects\PycharmProjects\unittest\common\request.py", line 9, in do_post
    return requests.post(url=url,data=data,headers=headers,**kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 530, in request
    resp = self.send(prep, **send_kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 643, in send
    r = adapter.send(request, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft1.5' class='none'>
        <td class='errorCase'><div class='testcase'>test_CaseInfo_00005: test_CaseInfo_00005</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.5')" >
            错误</a>

        <div id='div_ft1.5' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.5').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft1.5: case1_5
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 444, in wrap_socket
    cnx.do_handshake()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1894, in do_handshake
    self._raise_ssl_error(self._ssl, result)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1632, in _raise_ssl_error
    _raise_current_error()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\_util.py", line 57, in exception_from_error_queue
    raise exception_type(errors)
OpenSSL.SSL.Error: [('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 594, in urlopen
    self._prepare_proxy(conn)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 815, in _prepare_proxy
    conn.connect()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connection.py", line 349, in connect
    self.sock = ssl_wrap_socket(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\ssl_.py", line 359, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 450, in wrap_socket
    raise ssl.SSLError('bad handshake: %r' % e)
ssl.SSLError: ("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])",)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 439, in send
    resp = conn.urlopen(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 637, in urlopen
    retries = retries.increment(method, url, error=e, _pool=self,
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\retry.py", line 398, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case1_案例与法规\test_案例.py", line 75, in test_CaseInfo
    response = ad.do_post(url,json.dumps(payloads),headers=headers)
  File "D:\Projects\PycharmProjects\unittest\common\request.py", line 9, in do_post
    return requests.post(url=url,data=data,headers=headers,**kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 530, in request
    resp = self.send(prep, **send_kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 643, in send
    r = adapter.send(request, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft1.6' class='none'>
        <td class='errorCase'><div class='testcase'>test_CaseInfo_00006: test_CaseInfo_00006</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.6')" >
            错误</a>

        <div id='div_ft1.6' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.6').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft1.6: case1_6
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 444, in wrap_socket
    cnx.do_handshake()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1894, in do_handshake
    self._raise_ssl_error(self._ssl, result)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\SSL.py", line 1632, in _raise_ssl_error
    _raise_current_error()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\OpenSSL\_util.py", line 57, in exception_from_error_queue
    raise exception_type(errors)
OpenSSL.SSL.Error: [('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 594, in urlopen
    self._prepare_proxy(conn)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 815, in _prepare_proxy
    conn.connect()
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connection.py", line 349, in connect
    self.sock = ssl_wrap_socket(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\ssl_.py", line 359, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\contrib\pyopenssl.py", line 450, in wrap_socket
    raise ssl.SSLError('bad handshake: %r' % e)
ssl.SSLError: ("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])",)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 439, in send
    resp = conn.urlopen(
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py", line 637, in urlopen
    retries = retries.increment(method, url, error=e, _pool=self,
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\util\retry.py", line 398, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case1_案例与法规\test_案例.py", line 75, in test_CaseInfo
    response = ad.do_post(url,json.dumps(payloads),headers=headers)
  File "D:\Projects\PycharmProjects\unittest\common\request.py", line 9, in do_post
    return requests.post(url=url,data=data,headers=headers,**kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 530, in request
    resp = self.send(prep, **send_kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\sessions.py", line 643, in send
    r = adapter.send(request, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\requests\adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /library/case/info (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt1.7' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_Cases_00001: test_Cases_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt1.7')" >
            通过</a>

        <div id='div_pt1.7' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt1.7').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt1.7: case4_1----搜索关键词“偷窃”
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回1:23
实际返回1:23
期望返回2:10
实际返回2:10


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt1.8' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_Cases_00002: test_Cases_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt1.8')" >
            通过</a>

        <div id='div_pt1.8' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt1.8').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt1.8: case4_2----搜索关键词“离婚”
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回1:4358
实际返回1:4366
期望返回2:10
实际返回2:10


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt1.9' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_Cases_00003: test_Cases_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt1.9')" >
            通过</a>

        <div id='div_pt1.9' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt1.9').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt1.9: case4_3----搜索关键词“程序员”
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回1:1
实际返回1:1
期望返回2:1
实际返回2:1


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

<tr class='passClass'>
    <td>Case.case1_案例与法规.test_法规.TestLawInfo</td>
    <td>10</td>
    <td>10</td>
    <td>0</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c2',10)">详情</a></td>
    <td>&nbsp;</td>
</tr>

    <tr id='pt2.1' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_LawInfo_00001: test_LawInfo_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt2.1')" >
            通过</a>

        <div id='div_pt2.1' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt2.1').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt2.1: case2_1----《交通法》目录章节数
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:1
实际返回:1


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt2.2' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_LawInfo_00002: test_LawInfo_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt2.2')" >
            通过</a>

        <div id='div_pt2.2' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt2.2').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt2.2: case2_2----《宪法》目录章节数
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:13
实际返回:13


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt2.3' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_LawInfo_00003: test_LawInfo_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt2.3')" >
            通过</a>

        <div id='div_pt2.3' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt2.3').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt2.3: case2_3----《婚姻法》目录章节数
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:6
实际返回:6


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt2.4' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_LawInfo_00004: test_LawInfo_00004</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt2.4')" >
            通过</a>

        <div id='div_pt2.4' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt2.4').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt2.4: case2_4----《关于交易法通知》目录章节数
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:0
实际返回:0


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt2.5' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_Laws_00001: test_Laws_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt2.5')" >
            通过</a>

        <div id='div_pt2.5' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt2.5').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt2.5: case3_1----搜索关键词“通知”，分页大小设为10，当前页设为1
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回1:140290
实际返回1:140499
期望返回2:10
实际返回2:10


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt2.6' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_Laws_00002: test_Laws_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt2.6')" >
            通过</a>

        <div id='div_pt2.6' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt2.6').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt2.6: case3_2----搜索关键词“详情”，分页大小设为10，当前页设为1
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回1:1
实际返回1:1
期望返回2:1
实际返回2:1


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt2.7' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_Laws_00003: test_Laws_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt2.7')" >
            通过</a>

        <div id='div_pt2.7' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt2.7').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt2.7: case3_3----搜索关键词“宪法”，分页大小设为10，当前页设为1
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回1:53
实际返回1:53
期望返回2:10
实际返回2:10


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt2.8' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_Laws_00004: test_Laws_00004</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt2.8')" >
            通过</a>

        <div id='div_pt2.8' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt2.8').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt2.8: case3_4----搜索关键词“宪法”，分页大小设为20，当前页设为1
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回1:53
实际返回1:53
期望返回2:20
实际返回2:20


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt2.9' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_Laws_00005: test_Laws_00005</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt2.9')" >
            通过</a>

        <div id='div_pt2.9' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt2.9').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt2.9: case3_5----搜索关键词“宪法”，分页大小设为10，当前页设为6
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回1:53
实际返回1:53
期望返回2:3
实际返回2:3


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt2.10' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_Laws_00006: test_Laws_00006</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt2.10')" >
            通过</a>

        <div id='div_pt2.10' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt2.10').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt2.10: case3_7----搜索关键词“宪法”，分页大小设为20，当前页设为3
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回1:53
实际返回1:53
期望返回2:13
实际返回2:13


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

<tr class='passClass'>
    <td>Case.case2_文书纠错.test_上诉法院信息.TestRepetition</td>
    <td>6</td>
    <td>6</td>
    <td>0</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c3',6)">详情</a></td>
    <td>&nbsp;</td>
</tr>

    <tr id='pt3.1' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_appellateCourt_00001: test_appellateCourt_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt3.1')" >
            通过</a>

        <div id='div_pt3.1' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt3.1').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt3.1: case11_1----登录账号最高法，上诉最高法
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1800, 'keyword': '最高人民法院', 'suggestion': None, 'tip': ''}
实际返回:{'level': 1, 'type': 1800, 'keyword': '最高人民法院', 'suggestion': None, 'tip': ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt3.2' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_appellateCourt_00002: test_appellateCourt_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt3.2')" >
            通过</a>

        <div id='div_pt3.2' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt3.2').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt3.2: case11_2----登录账号“福建省厦门中级人民法院”，上诉“最高人民法院”
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1800, 'keyword': '最高人民法院', 'suggestion': '福建省高级人民法院', 'tip': ''}
实际返回:{'level': 1, 'type': 1800, 'keyword': '最高人民法院', 'suggestion': '福建省高级人民法院', 'tip': ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt3.3' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_appellateCourt_00003: test_appellateCourt_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt3.3')" >
            通过</a>

        <div id='div_pt3.3' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt3.3').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt3.3: case11_3----登录账号“延庆县人民法院”，上诉“最高人民法院”
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1800, 'keyword': '最高人民法院', 'suggestion': '北京市第二中级人民法院', 'tip': ''}
实际返回:{'level': 1, 'type': 1800, 'keyword': '最高人民法院', 'suggestion': '北京市第二中级人民法院', 'tip': ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt3.4' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_appellateCourt_00004: test_appellateCourt_00004</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt3.4')" >
            通过</a>

        <div id='div_pt3.4' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt3.4').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt3.4: case11_4----登录账号“遵化市人民法院”，上诉“河北省高级人民法院”
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1800, 'keyword': '河北省高级人民法院', 'suggestion': '河北省唐山市中级人民法院', 'tip': ''}
实际返回:{'level': 1, 'type': 1800, 'keyword': '河北省高级人民法院', 'suggestion': '河北省唐山市中级人民法院', 'tip': ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt3.5' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_appellateCourt_00005: test_appellateCourt_00005</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt3.5')" >
            通过</a>

        <div id='div_pt3.5' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt3.5').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt3.5: case11_3----登录账号“中国人民解放军北京军区河北军事法院”，上诉“中国人民解放军北京军区河北军事法院”
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1800, 'keyword': '中国人民解放军北京军区河北军事法院', 'suggestion': '中国人民解放军北京军区军事法院', 'tip': ''}
实际返回:{'level': 1, 'type': 1800, 'keyword': '中国人民解放军北京军区河北军事法院', 'suggestion': '中国人民解放军北京军区军事法院', 'tip': ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt3.6' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_appellateCourt_00006: test_appellateCourt_00006</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt3.6')" >
            通过</a>

        <div id='div_pt3.6' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt3.6').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt3.6: case11_6----登录账号最高法，终审判决
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

<tr class='failClass'>
    <td>Case.case2_文书纠错.test_信息重复.TestRepetition</td>
    <td>8</td>
    <td>6</td>
    <td>2</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c4',8)">详情</a></td>
    <td>&nbsp;</td>
</tr>

    <tr id='pt4.1' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_repetition_00001: test_repetition_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt4.1')" >
            通过</a>

        <div id='div_pt4.1' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt4.1').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt4.1: case10_1----数字重复
{'judgeId': 5, 'paragraphs': [{'index': 2, 'content': '2020'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt4.2' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_repetition_00002: test_repetition_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt4.2')" >
            通过</a>

        <div id='div_pt4.2' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt4.2').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt4.2: case10_2----一段里面出现多个相同数字
{'judgeId': 5, 'paragraphs': [{'index': 2, 'content': '被告人名下工商银行开户账号银行卡号6222082502004841432，以上财产价值暂计：1，560,000.00元人民币； 3.请求法院分割夫妻共同财产被告李四借给张金彩750,000.00元人民币；4.请求法院分割被告在云南恒久科技有限公司50%的股权， 价值约300,000.00元人民币；5.由被告李四一次性补偿原告张三200,000.00元人民币；6.由被告承担本案诉讼费用。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt4.3' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_repetition_00003: test_repetition_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt4.3')" >
            通过</a>

        <div id='div_pt4.3' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt4.3').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt4.3: case10_3----文字重复
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '《中《中华人民共和国民法通则》'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[{'level': 1, 'type': 1700, 'keyword': '《中《中', 'suggestion': '', 'tip': None}]
实际返回:[{'level': 1, 'type': 1700, 'keyword': '《中《中', 'suggestion': '', 'tip': None}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft4.4' class='none'>
        <td class='failCase'><div class='testcase'>test_repetition_00004: test_repetition_00004</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.4')" >
            失败</a>

        <div id='div_ft4.4' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.4').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft4.4: case10_4----段落重复
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '根据当事人的陈述、举证及庭审查明的事实，本院确认如下事实。'}, {'index': 3, 'content': '根据当事人的陈述、举证及庭审查明的事实，本院确认如下事实。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[{'level': 1, 'type': 1700, 'keyword': '根据当事人的陈述、举证及庭审查明的事实，本院确认如下事实。', 'suggestion': '', 'tip': None}, {'level': 1, 'type': 1700, 'keyword': '根据当事人的陈述、举证及庭审查明的事实，本院确认如下事实。', 'suggestion': '', 'tip': None}]
实际返回:[]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_信息重复.py", line 43, in test_repetition
    res = response.json()["detections"][0]["corrections"]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_信息重复.py", line 50, in test_repetition
    self.assertEqual(res, expectResult)
AssertionError: Lists differ: [] != [{'level': 1, 'type': 1700, 'keyword': '根据[159 chars]one}]

Second list contains 2 additional elements.
First extra element 0:
{'level': 1, 'type': 1700, 'keyword': '根据当事人的陈述、举证及庭审查明的事实，本院确认如下事实。', 'suggestion': '', 'tip': None}

- []
+ [{'keyword': '根据当事人的陈述、举证及庭审查明的事实，本院确认如下事实。',
+   'level': 1,
+   'suggestion': '',
+   'tip': None,
+   'type': 1700},
+  {'keyword': '根据当事人的陈述、举证及庭审查明的事实，本院确认如下事实。',
+   'level': 1,
+   'suggestion': '',
+   'tip': None,
+   'type': 1700}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft4.5' class='none'>
        <td class='failCase'><div class='testcase'>test_repetition_00005: test_repetition_00005</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.5')" >
            失败</a>

        <div id='div_ft4.5' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.5').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft4.5: case10_5----文字重复
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉族，汉族，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[{'level': 1, 'type': 1700, 'keyword': '，汉族', 'suggestion': '', 'tip': None}]
实际返回:[{'level': 1, 'type': 1700, 'keyword': '，汉族，汉族', 'suggestion': '', 'tip': None}, {'level': 1, 'type': 1700, 'keyword': '汉族', 'suggestion': '', 'tip': None}]
期望返回:[{'level': 1, 'type': 1700, 'keyword': '，汉族', 'suggestion': '', 'tip': None}]
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1700, 'keyword': '，汉族，汉族', 'suggestion': '', 'tip': None}, {'level': 1, 'type': 1700, 'keyword': '汉族', 'suggestion': '', 'tip': None}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_信息重复.py", line 45, in test_repetition
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: Lists differ: [{'le[33 chars] '，汉族，汉族', 'suggestion': '', 'tip': None}, {'l[67 chars]one}] != [{'le[33 chars] '，汉族', 'suggestion': '', 'tip': None}]

First differing element 0:
{'level': 1, 'type': 1700, 'keyword': '，汉族，汉族', 'suggestion': '', 'tip': None}
{'level': 1, 'type': 1700, 'keyword': '，汉族', 'suggestion': '', 'tip': None}

First list contains 1 additional elements.
First extra element 1:
{'level': 1, 'type': 1700, 'keyword': '汉族', 'suggestion': '', 'tip': None}

- [{'keyword': '，汉族，汉族', 'level': 1, 'suggestion': '', 'tip': None, 'type': 1700},
-  {'keyword': '汉族', 'level': 1, 'suggestion': '', 'tip': None, 'type': 1700}]
? ^

+ [{'keyword': '，汉族', 'level': 1, 'suggestion': '', 'tip': None, 'type': 1700}]
? ^             +


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_信息重复.py", line 50, in test_repetition
    self.assertEqual(res, expectResult)
AssertionError: Lists differ: [{'index': 2, 'corrections': [{'level': 1, '[138 chars]e}]}] != [{'level': 1, 'type': 1700, 'keyword': '，汉族'[28 chars]one}]

First differing element 0:
{'index': 2, 'corrections': [{'level': 1, '[137 chars]ne}]}
{'level': 1, 'type': 1700, 'keyword': '，汉族'[27 chars]None}

+ [{'keyword': '，汉族', 'level': 1, 'suggestion': '', 'tip': None, 'type': 1700}]
- [{'corrections': [{'keyword': '，汉族，汉族',
-                    'level': 1,
-                    'suggestion': '',
-                    'tip': None,
-                    'type': 1700},
-                   {'keyword': '汉族',
-                    'level': 1,
-                    'suggestion': '',
-                    'tip': None,
-                    'type': 1700}],
-   'index': 2}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt4.6' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_repetition_00006: test_repetition_00006</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt4.6')" >
            通过</a>

        <div id='div_pt4.6' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt4.6').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt4.6: case10_6----叠词-AABB型
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '口口声声'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt4.7' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_repetition_00007: test_repetition_00007</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt4.7')" >
            通过</a>

        <div id='div_pt4.7' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt4.7').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt4.7: case10_7----叠词-ABAB型
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '一个一个'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[{'level': 1, 'type': 1700, 'keyword': '一个一个', 'suggestion': '', 'tip': None}]
实际返回:[{'level': 1, 'type': 1700, 'keyword': '一个一个', 'suggestion': '', 'tip': None}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt4.8' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_repetition_00008: test_repetition_00008</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt4.8')" >
            通过</a>

        <div id='div_pt4.8' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt4.8').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt4.8: case10_8----段落里面有语句重复
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告张三起诉称：1.依法判令撤销原被告双方签订的协议（财产分割部分）； 2.依法分割夫妻共同财产车辆：云A656Y1大众朗逸轿车一辆、云A656Y1奥迪Q5越野车一辆；不动产： a、昆明市盘龙区青云街道办事处东三环西侧金色交响家园，房地产权证编号：房地证津字第52号； b、昆明市盘龙区青云街道办事处东三环西侧金色交响家园；'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[{'level': 1, 'type': 1700, 'keyword': '昆明市盘龙区青云街道办事处东三环西侧金色交响家园', 'suggestion': '', 'tip': None}]
实际返回:[{'level': 1, 'type': 1700, 'keyword': '昆明市盘龙区青云街道办事处东三环西侧金色交响家园', 'suggestion': '', 'tip': None}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

<tr class='passClass'>
    <td>Case.case2_文书纠错.test_受理费承担人.TestFees</td>
    <td>8</td>
    <td>8</td>
    <td>0</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c5',8)">详情</a></td>
    <td>&nbsp;</td>
</tr>

    <tr id='pt5.1' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_fees_00001: test_fees_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt5.1')" >
            通过</a>

        <div id='div_pt5.1' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt5.1').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt5.1: case12_1----有承担人。
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt5.2' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_fees_00002: test_fees_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt5.2')" >
            通过</a>

        <div id='div_pt5.2' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt5.2').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt5.2: case12_2----只有名字，没有诉讼地位
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1803, 'keyword': '', 'suggestion': '', 'tip': '受理费承担人须指定当事人身份'}
实际返回:{'level': 1, 'type': 1803, 'keyword': '', 'suggestion': '', 'tip': '受理费承担人须指定当事人身份'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt5.3' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_fees_00003: test_fees_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt5.3')" >
            通过</a>

        <div id='div_pt5.3' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt5.3').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt5.3: case12_3----有多名承担人
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt5.4' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_fees_00004: test_fees_00004</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt5.4')" >
            通过</a>

        <div id='div_pt5.4' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt5.4').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt5.4: case12_4----上诉部分也有受理费信息
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt5.5' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_fees_00005: test_fees_00005</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt5.5')" >
            通过</a>

        <div id='div_pt5.5' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt5.5').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt5.5: case12_5----遗漏承担人信息
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1802, 'keyword': '', 'suggestion': '', 'tip': '请补充受理费并指定承担人'}
实际返回:{'level': 1, 'type': 1802, 'keyword': '', 'suggestion': '', 'tip': '请补充受理费并指定承担人'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt5.6' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_fees_00006: test_fees_00006</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt5.6')" >
            通过</a>

        <div id='div_pt5.6' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt5.6').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt5.6: case12_6----有附加费用但没有承担人
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1802, 'keyword': '', 'suggestion': '', 'tip': '请补充受理费并指定承担人'}
实际返回:{'level': 1, 'type': 1802, 'keyword': '', 'suggestion': '', 'tip': '请补充受理费并指定承担人'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt5.7' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_fees_00007: test_fees_00007</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt5.7')" >
            通过</a>

        <div id='div_pt5.7' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt5.7').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt5.7: case12_7----有减收费用但没有承担人
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1802, 'keyword': '', 'suggestion': '', 'tip': '请补充受理费并指定承担人'}
实际返回:{'level': 1, 'type': 1802, 'keyword': '', 'suggestion': '', 'tip': '请补充受理费并指定承担人'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt5.8' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_fees_00008: test_fees_00008</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt5.8')" >
            通过</a>

        <div id='div_pt5.8' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt5.8').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt5.8: case12_8----遗漏整段受理费信息
https://localhost:5001/Judgements/ErrorDetect
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1802, 'keyword': '', 'suggestion': '', 'tip': '请补充受理费并指定承担人'}
实际返回:{'level': 1, 'type': 1802, 'keyword': '', 'suggestion': '', 'tip': '请补充受理费并指定承担人'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

<tr class='failClass'>
    <td>Case.case2_文书纠错.test_地区套用错误.TestAreaError</td>
    <td>11</td>
    <td>7</td>
    <td>4</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c6',11)">详情</a></td>
    <td>&nbsp;</td>
</tr>

    <tr id='pt6.1' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_AreaError_00001: test_AreaError_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt6.1')" >
            通过</a>

        <div id='div_pt6.1' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt6.1').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt6.1: case7_1----省与市不匹配
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1600, 'keyword': '福建省九江市', 'suggestion': '', 'tip': None}
实际返回:{'level': 1, 'type': 1600, 'keyword': '福建省九江市', 'suggestion': '', 'tip': None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt6.2' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_AreaError_00002: test_AreaError_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt6.2')" >
            通过</a>

        <div id='div_pt6.2' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt6.2').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt6.2: case7_2----省与市匹配
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt6.3' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_AreaError_00003: test_AreaError_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt6.3')" >
            通过</a>

        <div id='div_pt6.3' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt6.3').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt6.3: case7_3----省与市不匹配
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1600, 'keyword': '广西省昆明市盘龙区', 'suggestion': '', 'tip': None}
实际返回:{'level': 1, 'type': 1600, 'keyword': '广西省昆明市盘龙区', 'suggestion': '', 'tip': None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft6.4' class='none'>
        <td class='failCase'><div class='testcase'>test_AreaError_00004: test_AreaError_00004</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft6.4')" >
            失败</a>

        <div id='div_ft6.4' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft6.4').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft6.4: case7_4----省下来直接到县或者县级市
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1600, 'keyword': '福建省漳浦县', 'suggestion': '', 'tip': None}
实际返回:{'level': 1, 'type': 1401, 'keyword': '.', 'suggestion': '。', 'tip': ''}
期望返回:{'level': 1, 'type': 1600, 'keyword': '福建省漳浦县', 'suggestion': '', 'tip': None}
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1401, 'keyword': '.', 'suggestion': '。', 'tip': ''}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_地区套用错误.py", line 46, in test_AreaError
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1401, 'keyword': '.', 'suggestion': '。', 'tip': ''} != {'level': 1, 'type': 1600, 'keyword': '福建省漳浦县', 'suggestion': '', 'tip': None}
- {'keyword': '.', 'level': 1, 'suggestion': '。', 'tip': '', 'type': 1401}
?              ^                              -          ^^           ^ ^

+ {'keyword': '福建省漳浦县', 'level': 1, 'suggestion': '', 'tip': None, 'type': 1600}
?              ^^^^^^                                        ^^^^           ^ ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_地区套用错误.py", line 51, in test_AreaError
    self.assertEqual(res, expectResult)
AssertionError: [{'index': 2, 'corrections': [{'level': 1[59 chars]'}]}] != {'level': 1, 'type': 1600, 'keyword': '福建[32 chars]None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft6.5' class='none'>
        <td class='failCase'><div class='testcase'>test_AreaError_00005: test_AreaError_00005</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft6.5')" >
            失败</a>

        <div id='div_ft6.5' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft6.5').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft6.5: case7_5----省下来直接到县或者县级市
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1600, 'keyword': '江苏省丹徒区', 'suggestion': '', 'tip': None}
实际返回:{'level': 1, 'type': 1401, 'keyword': '"', 'suggestion': '', 'tip': '“”'}
期望返回:{'level': 1, 'type': 1600, 'keyword': '江苏省丹徒区', 'suggestion': '', 'tip': None}
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1401, 'keyword': '"', 'suggestion': '', 'tip': '“”'}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_地区套用错误.py", line 46, in test_AreaError
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1401, 'keyword': '"', 'suggestion': '', 'tip': '“”'} != {'level': 1, 'type': 1600, 'keyword': '江苏省丹徒区', 'suggestion': '', 'tip': None}
- {'keyword': '"', 'level': 1, 'suggestion': '', 'tip': '“”', 'type': 1401}
?              ^                                        ^^^^           ^ ^

+ {'keyword': '江苏省丹徒区', 'level': 1, 'suggestion': '', 'tip': None, 'type': 1600}
?              ^^^^^^                                        ^^^^           ^ ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_地区套用错误.py", line 51, in test_AreaError
    self.assertEqual(res, expectResult)
AssertionError: [{'index': 2, 'corrections': [{'level': 1[60 chars]'}]}] != {'level': 1, 'type': 1600, 'keyword': '江苏[32 chars]None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft6.6' class='none'>
        <td class='failCase'><div class='testcase'>test_AreaError_00006: test_AreaError_00006</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft6.6')" >
            失败</a>

        <div id='div_ft6.6' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft6.6').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft6.6: case7_6----省下来直接到县或者县级市
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1600, 'keyword': '福建省龙海市', 'suggestion': '', 'tip': None}
实际返回:[]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_地区套用错误.py", line 44, in test_AreaError
    res = response.json()["detections"][0]["corrections"][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_地区套用错误.py", line 51, in test_AreaError
    self.assertEqual(res, expectResult)
AssertionError: [] != {'level': 1, 'type': 1600, 'keyword': '福建省龙海市', 'suggestion': '', 'tip': None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt6.7' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_AreaError_00007: test_AreaError_00007</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt6.7')" >
            通过</a>

        <div id='div_pt6.7' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt6.7').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt6.7: case7_7----直辖市与县
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt6.8' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_AreaError_00008: test_AreaError_00008</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt6.8')" >
            通过</a>

        <div id='div_pt6.8' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt6.8').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt6.8: case7_8----直辖市与县
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft6.9' class='none'>
        <td class='failCase'><div class='testcase'>test_AreaError_00009: test_AreaError_00009</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft6.9')" >
            失败</a>

        <div id='div_ft6.9' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft6.9').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft6.9: case7_9----非地区套用的
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:{'level': 1, 'type': 1600, 'keyword': '北京市隆安（南京）律师事务所律师', 'suggestion': '', 'tip': None}
期望返回:[]
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1600, 'keyword': '北京市隆安（南京）律师事务所律师', 'suggestion': '', 'tip': None}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_地区套用错误.py", line 46, in test_AreaError
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1600, 'keyword': '北京[42 chars]None} != []

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_地区套用错误.py", line 51, in test_AreaError
    self.assertEqual(res, expectResult)
AssertionError: Lists differ: [{'index': 2, 'corrections': [{'level': 1,[74 chars]e}]}] != []

First list contains 1 additional elements.
First extra element 0:
{'index': 2, 'corrections': [{'level': 1, 'type': 1600, 'keyword': '北京市隆安（南京）律师事务所律师', 'suggestion': '', 'tip': None}]}

+ []
- [{'corrections': [{'keyword': '北京市隆安（南京）律师事务所律师',
-                    'level': 1,
-                    'suggestion': '',
-                    'tip': None,
-                    'type': 1600}],
-   'index': 2}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt6.10' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_AreaError_00010: test_AreaError_00010</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt6.10')" >
            通过</a>

        <div id='div_pt6.10' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt6.10').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt6.10: case7_10----非地区套用的
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt6.11' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_AreaError_00011: test_AreaError_00011</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt6.11')" >
            通过</a>

        <div id='div_pt6.11' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt6.11').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt6.11: case7_10----非地区套用的
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

<tr class='failClass'>
    <td>Case.case2_文书纠错.test_当事人信息.TestParty</td>
    <td>24</td>
    <td>15</td>
    <td>9</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c7',24)">详情</a></td>
    <td>&nbsp;</td>
</tr>

    <tr id='pt7.1' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00001: test_party_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.1')" >
            通过</a>

        <div id='div_pt7.1' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.1').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.1: case6_1----顺序错误
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，汉族，男，系文盲，1980年11月16日生，汉族，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，汉族，男，系文盲，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，汉族，男，系文盲，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.2' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00002: test_party_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.2')" >
            通过</a>

        <div id='div_pt7.2' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.2').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.2: case6_2----当事人信息正确（包含职业）
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉族，农村务农，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.3' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00003: test_party_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.3')" >
            通过</a>

        <div id='div_pt7.3' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.3').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.3: case6_3----当事人信息正确（不包含职业）
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉族，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.4' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00004: test_party_00004</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.4')" >
            通过</a>

        <div id='div_pt7.4' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.4').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.4: case6_4----民族信息缺少“族”
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，男，1980年11月16日生，汉，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '民族后缺少“族”字'}
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，男，1980年11月16日生，汉，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '民族后缺少“族”字'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft7.5' class='none'>
        <td class='failCase'><div class='testcase'>test_party_00005: test_party_00005</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.5')" >
            失败</a>

        <div id='div_ft7.5' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.5').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft7.5: case6_5----“住所信息”不规范-居住地
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉族，居住地云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，男，1980年11月16日生，汉族，居住地云南省昆明市盘龙区', 'suggestion': '', 'tip': '地址前缺少“住”字'}
实际返回:[]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 45, in test_party
    res = response.json()["detections"][0]["corrections"][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 52, in test_party
    self.assertEqual(res, expectResult)
AssertionError: [] != {'level': 1, 'type': 1101, 'keyword': '原告[69 chars]住”字'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.6' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00006: test_party_00006</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.6')" >
            通过</a>

        <div id='div_pt7.6' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.6').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.6: case6_6----“住所信息”不规范-住址
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉族，住址云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft7.7' class='none'>
        <td class='failCase'><div class='testcase'>test_party_00007: test_party_00007</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.7')" >
            失败</a>

        <div id='div_ft7.7' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.7').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft7.7: case6_7----“住所信息”不规范-联系地址
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉族，联系住址云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，男，1980年11月16日生，汉族，联系住址云南省昆明市盘龙区', 'suggestion': '', 'tip': '地址前缺少“住”字'}
实际返回:[]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 45, in test_party
    res = response.json()["detections"][0]["corrections"][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 52, in test_party
    self.assertEqual(res, expectResult)
AssertionError: [] != {'level': 1, 'type': 1101, 'keyword': '原告[70 chars]住”字'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.8' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00008: test_party_00008</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.8')" >
            通过</a>

        <div id='div_pt7.8' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.8').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.8: case6_8----“住所信息”不规范-缺少“住”
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉族，云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，男，1980年11月16日生，汉族，云南省昆明市盘龙区', 'suggestion': '', 'tip': '地址前缺少“住”字'}
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，男，1980年11月16日生，汉族，云南省昆明市盘龙区', 'suggestion': '', 'tip': '地址前缺少“住”字'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft7.9' class='none'>
        <td class='failCase'><div class='testcase'>test_party_00009: test_party_00009</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.9')" >
            失败</a>

        <div id='div_ft7.9' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.9').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft7.9: case6_9----当事人信息不完整-缺少民族信息
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1105, 'keyword': '原告：张三，男，1980年11月16日生，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，男，1980年11月16日生，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '民族后缺少“族”字'}
期望返回:{'level': 1, 'type': 1105, 'keyword': '原告：张三，男，1980年11月16日生，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1101, 'keyword': '原告：张三，男，1980年11月16日生，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '民族后缺少“族”字'}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 47, in test_party
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'lev[14 chars]: 1101, 'keyword': '原告：张三，男，1980年11月16日生，住云南省昆[40 chars]族”字'} != {'lev[14 chars]: 1105, 'keyword': '原告：张三，男，1980年11月16日生，住云南省昆[95 chars]住址]'}
  {'keyword': '原告：张三，男，1980年11月16日生，住云南省昆明市盘龙区',
   'level': 1,
   'suggestion': '',
-  'tip': '民族后缺少“族”字',
+  'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]',
-  'type': 1101}
?             ^

+  'type': 1105}
?             ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 52, in test_party
    self.assertEqual(res, expectResult)
AssertionError: [{'index': 2, 'corrections': [{'level': 1[97 chars]'}]}] != {'level': 1, 'type': 1105, 'keyword': '原告[119 chars]住址]'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.10' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00010: test_party_00010</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.10')" >
            通过</a>

        <div id='div_pt7.10' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.10').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.10: case6_10----当事人信息不完整-缺少性别信息
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，1980年11月16日生，汉族，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '请写明当事人性别'}
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '请写明当事人性别'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.11' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00011: test_party_00011</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.11')" >
            通过</a>

        <div id='div_pt7.11' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.11').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.11: case6_11----当事人信息不完整-出生日期不完整
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月生，汉族，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，男，1980年11月生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，男，1980年11月生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.12' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00012: test_party_00012</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.12')" >
            通过</a>

        <div id='div_pt7.12' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.12').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.12: case6_12----当事人信息不完整-缺少姓名
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：，男，1980年11月16日生，汉族，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告：，男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告：，男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft7.13' class='none'>
        <td class='failCase'><div class='testcase'>test_party_00013: test_party_00013</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.13')" >
            失败</a>

        <div id='div_ft7.13' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.13').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft7.13: case6_13----当事人信息不完整-缺少姓名
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告，男，1980年11月16日生，汉族，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告，男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告，男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告，男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1101, 'keyword': '原告，男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 47, in test_party
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'lev[89 chars]p': '诉讼地位与姓名之间缺少冒号'} != {'lev[89 chars]p': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民[20 chars]住址]'}
  {'keyword': '原告，男，1980年11月16日生，汉族，住云南省昆明市盘龙区',
   'level': 1,
   'suggestion': '',
-  'tip': '诉讼地位与姓名之间缺少冒号',
+  'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]',
   'type': 1101}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 52, in test_party
    self.assertEqual(res, expectResult)
AssertionError: [{'index': 2, 'corrections': [{'level': 1[101 chars]'}]}] != {'level': 1, 'type': 1101, 'keyword': '原告[119 chars]住址]'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.14' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00014: test_party_00014</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.14')" >
            通过</a>

        <div id='div_pt7.14' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.14').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.14: case6_14----当事人信息不完整-诉讼地位与姓名之间没有冒号
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告张三，男，1980年11月16日生，汉族，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告张三，男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告张三，男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft7.15' class='none'>
        <td class='failCase'><div class='testcase'>test_party_00015: test_party_00015</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.15')" >
            失败</a>

        <div id='div_ft7.15' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.15').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft7.15: case6_15----当事人信息不完整-缺少诉讼地位
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '张三，男，1980年11月16日生，汉族，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1106, 'keyword': '张三，男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}
实际返回:[]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 45, in test_party
    res = response.json()["detections"][0]["corrections"][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 52, in test_party
    self.assertEqual(res, expectResult)
AssertionError: [] != {'level': 1, 'type': 1106, 'keyword': '张三[119 chars]住址]'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.16' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00016: test_party_00016</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.16')" >
            通过</a>

        <div id='div_pt7.16' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.16').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.16: case6_16----当事人信息错误-添加不必要信息
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，性别男，1980年11月16日生，汉族，住云南省昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，性别男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告：张三，性别男，1980年11月16日生，汉族，住云南省昆明市盘龙区', 'suggestion': '', 'tip': '[诉讼地位]：[姓名]，[性别]，[出生年份]年[出生月份]月[出生日]日生，[民族]族，[职业或者工作单位和职务]，住[住址]'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.17' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00017: test_party_00017</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.17')" >
            通过</a>

        <div id='div_pt7.17' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.17').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.17: case6_17----当事人信息不对等
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉族，住云南省昆明市盘龙区。 被告：李四，男，1980年11月16日生，汉族，住昆明市盘龙区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 2, 'type': 1102, 'keyword': '', 'suggestion': '', 'tip': None}
实际返回:{'level': 2, 'type': 1102, 'keyword': '', 'suggestion': '', 'tip': None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.18' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00018: test_party_00018</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.18')" >
            通过</a>

        <div id='div_pt7.18' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.18').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.18: case6_18----当事人信息错误-住所地址不同时未写明属于哪个省份
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉族，住昆明市盘龙区。 被告：李四，男，1980年11月16日生，汉族，住厦门市思明区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 2, 'type': 1103, 'keyword': '', 'suggestion': '', 'tip': None}
实际返回:{'level': 2, 'type': 1103, 'keyword': '', 'suggestion': '', 'tip': None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.19' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00019: test_party_00019</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.19')" >
            通过</a>

        <div id='div_pt7.19' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.19').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.19: case6_19----当事人信息信息对等
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：张三，男，1980年11月16日生，汉族，住厦门市集美区。'}, {'index': 3, 'content': '被告：李四，男，1980年11月16日生，汉族，住厦门市思明区。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft7.20' class='none'>
        <td class='failCase'><div class='testcase'>test_party_00020: test_party_00020</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.20')" >
            失败</a>

        <div id='div_ft7.20' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.20').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft7.20: case6_20----非当事人信息误报（【诉讼地位】，【日期】，【性别】）
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '被告于1992年10月21日晚上回到家中，跟住在隔壁的男人大打出手。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:{'level': 1, 'type': 1101, 'keyword': '被告于1992年10月21日晚上回到家中，跟住在隔壁的男人大打出手', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}
期望返回:[]
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1101, 'keyword': '被告于1992年10月21日晚上回到家中，跟住在隔壁的男人大打出手', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 47, in test_party
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1101, 'keyword': '被告[70 chars]少冒号'} != []

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 52, in test_party
    self.assertEqual(res, expectResult)
AssertionError: Lists differ: [{'index': 2, 'corrections': [{'level': 1,[102 chars]'}]}] != []

First list contains 1 additional elements.
First extra element 0:
{'index': 2, 'corrections': [{'level': 1, 'type': 1101, 'keyword': '被告于1992年10月21日晚上回到家中，跟住在隔壁的男人大打出手', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}]}

+ []
- [{'corrections': [{'keyword': '被告于1992年10月21日晚上回到家中，跟住在隔壁的男人大打出手',
-                    'level': 1,
-                    'suggestion': '',
-                    'tip': '诉讼地位与姓名之间缺少冒号',
-                    'type': 1101}],
-   'index': 2}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft7.21' class='none'>
        <td class='failCase'><div class='testcase'>test_party_00021: test_party_00021</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.21')" >
            失败</a>

        <div id='div_ft7.21' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.21').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft7.21: case6_21----非当事人信息误报（【诉讼地位】，【日期】，【性别】）
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '本院经审理认定事实如下：原告王德范系王志韬（已故）父亲、张素云系王志韬母亲， 王某1（2002年3月13日出生）系王志韬之子、王某2（2006年9月1日出生）系王志韬之女'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:{'level': 1, 'type': 1101, 'keyword': '本院经审理认定事实如下：原告王德范系王志韬（已故）父亲、张素云系王志韬母亲，王某1（2002年3月13日出生）系王志韬之子、王某2（2006年9月1日出生）系王志韬之女', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}
期望返回:[]
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1101, 'keyword': '本院经审理认定事实如下：原告王德范系王志韬（已故）父亲、张素云系王志韬母亲，王某1（2002年3月13日出生）系王志韬之子、王某2（2006年9月1日出生）系王志韬之女', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 47, in test_party
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1101, 'keyword': '本院[121 chars]少冒号'} != []

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 52, in test_party
    self.assertEqual(res, expectResult)
AssertionError: Lists differ: [{'index': 2, 'corrections': [{'level': 1,[153 chars]'}]}] != []

First list contains 1 additional elements.
First extra element 0:
{'index': 2, 'corrections': [{'level': 1, 'type': 1101, 'keyword': '本院经审理认定事实如下：原告王德范系王志韬（已故）父亲、张素云系王志韬母亲，王某1（2002年3月13日出生）系王志韬之子、王某2（2006年9月1日出生）系王志韬之女', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}]}

+ []
- [{'corrections': [{'keyword': '本院经审理认定事实如下：原告王德范系王志韬（已故）父亲、张素云系王志韬母亲，王某1（2002年3月13日出生）系王志韬之子、王某2（2006年9月1日出生）系王志韬之女',
-                    'level': 1,
-                    'suggestion': '',
-                    'tip': '诉讼地位与姓名之间缺少冒号',
-                    'type': 1101}],
-   'index': 2}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft7.22' class='none'>
        <td class='failCase'><div class='testcase'>test_party_00022: test_party_00022</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.22')" >
            失败</a>

        <div id='div_ft7.22' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.22').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft7.22: case6_22----非当事人信息误报（【诉讼地位】，【日期】，【汉】）
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告王德范、张素云、王某1、王某2与被告沈阳汉爵装饰材料有限公司劳动争议纠纷一案， 本院于2019年7月2日立案受理后，依法由审判员刘晓强担任审判长并主审，与人民陪审员李丞、孙维维组成合议庭，公开开庭进行了审理。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:{'level': 1, 'type': 1101, 'keyword': '原告王德范、张素云、王某1、王某2与被告沈阳汉爵装饰材料有限公司劳动争议纠纷一案，本院于2019年7月2日立案受理后，依法由审判员刘晓强担任审判长并主审，与人民陪审员李丞、孙维维组成合议庭，公开开庭进行了审理', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}
期望返回:[]
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1101, 'keyword': '原告王德范、张素云、王某1、王某2与被告沈阳汉爵装饰材料有限公司劳动争议纠纷一案，本院于2019年7月2日立案受理后，依法由审判员刘晓强担任审判长并主审，与人民陪审员李丞、孙维维组成合议庭，公开开庭进行了审理', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 47, in test_party
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1101, 'keyword': '原告[141 chars]少冒号'} != []

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 52, in test_party
    self.assertEqual(res, expectResult)
AssertionError: Lists differ: [{'index': 2, 'corrections': [{'level': 1,[173 chars]'}]}] != []

First list contains 1 additional elements.
First extra element 0:
{'index': 2, 'corrections': [{'level': 1, 'type': 1101, 'keyword': '原告王德范、张素云、王某1、王某2与被告沈阳汉爵装饰材料有限公司劳动争议纠纷一案，本院于2019年7月2日立案受理后，依法由审判员刘晓强担任审判长并主审，与人民陪审员李丞、孙维维组成合议庭，公开开庭进行了审理', 'suggestion': '', 'tip': '诉讼地位与姓名之间缺少冒号'}]}

+ []
- [{'corrections': [{'keyword': '原告王德范、张素云、王某1、王某2与被告沈阳汉爵装饰材料有限公司劳动争议纠纷一案，本院于2019年7月2日立案受理后，依法由审判员刘晓强担任审判长并主审，与人民陪审员李丞、孙维维组成合议庭，公开开庭进行了审理',
-                    'level': 1,
-                    'suggestion': '',
-                    'tip': '诉讼地位与姓名之间缺少冒号',
-                    'type': 1101}],
-   'index': 2}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft7.23' class='none'>
        <td class='failCase'><div class='testcase'>test_party_00023: test_party_00023</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.23')" >
            失败</a>

        <div id='div_ft7.23' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.23').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft7.23: case6_23----多名原告同时引用不需要都加诉讼地位
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：王德范，男，1947年12月24日出生，汉族，住址：沈阳市沈河区。'}, {'index': 3, 'content': '原告：张素云，女，1951年4月20日出生，汉族，住址：沈阳市沈河区。。'}, {'index': 4, 'content': '原告王德范、张素云，原告王某1、王某2法定代理人魏娇及四原告委托诉讼代理人王海峰、詹政，被告委托诉讼代理人赵春涛到庭参加诉讼。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:{'level': 1, 'type': 1106, 'keyword': '张素云', 'suggestion': '原告张素云', 'tip': None}
期望返回:[]
实际返回:[{'index': 4, 'corrections': [{'level': 1, 'type': 1106, 'keyword': '张素云', 'suggestion': '原告张素云', 'tip': None}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 47, in test_party
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1106, 'keyword': '张素云', 'suggestion': '原告张素云', 'tip': None} != []

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_当事人信息.py", line 52, in test_party
    self.assertEqual(res, expectResult)
AssertionError: Lists differ: [{'index': 4, 'corrections': [{'level': 1,[66 chars]e}]}] != []

First list contains 1 additional elements.
First extra element 0:
{'index': 4, 'corrections': [{'level': 1, 'type': 1106, 'keyword': '张素云', 'suggestion': '原告张素云', 'tip': None}]}

+ []
- [{'corrections': [{'keyword': '张素云',
-                    'level': 1,
-                    'suggestion': '原告张素云',
-                    'tip': None,
-                    'type': 1106}],
-   'index': 4}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt7.24' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_party_00024: test_party_00024</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt7.24')" >
            通过</a>

        <div id='div_pt7.24' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt7.24').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt7.24: case6_24----多名原告同时引用不需要都加诉讼地位
{'judgeId': 3, 'paragraphs': [{'index': 2, 'content': '原告：谢德云，男，1962年5月29日出生，汉族，，住如皋市九华镇马桥村8组3号。'}, {'index': 10, 'content': '案件受理费4,676元，由原告林印党负担。'}]}
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

<tr class='failClass'>
    <td>Case.case2_文书纠错.test_标点符号.TestPunctuation</td>
    <td>10</td>
    <td>7</td>
    <td>3</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c8',10)">详情</a></td>
    <td>&nbsp;</td>
</tr>

    <tr id='pt8.1' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_punctuation_00001: test_punctuation_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt8.1')" >
            通过</a>

        <div id='div_pt8.1' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt8.1').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt8.1: case7_1----中英文标点使用混乱-冒号错误
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1401, 'keyword': ':', 'suggestion': '：', 'tip': ''}
实际返回:{'level': 1, 'type': 1401, 'keyword': ':', 'suggestion': '：', 'tip': ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt8.2' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_punctuation_00002: test_punctuation_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt8.2')" >
            通过</a>

        <div id='div_pt8.2' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt8.2').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt8.2: case7_2----中英文标点使用混乱-小括号错误
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1401, 'keyword': '(', 'suggestion': '（', 'tip': ''}
实际返回:{'level': 1, 'type': 1401, 'keyword': '(', 'suggestion': '（', 'tip': ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt8.3' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_punctuation_00003: test_punctuation_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt8.3')" >
            通过</a>

        <div id='div_pt8.3' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt8.3').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt8.3: case7_3----中英文标点使用混乱-逗号错误
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1401, 'keyword': ',', 'suggestion': '，', 'tip': ''}
实际返回:{'level': 1, 'type': 1401, 'keyword': ',', 'suggestion': '，', 'tip': ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt8.4' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_punctuation_00004: test_punctuation_00004</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt8.4')" >
            通过</a>

        <div id='div_pt8.4' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt8.4').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt8.4: case7_4----中英文标点使用混乱-句号错误
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1401, 'keyword': '.', 'suggestion': '。', 'tip': ''}
实际返回:{'level': 1, 'type': 1401, 'keyword': '.', 'suggestion': '。', 'tip': ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt8.5' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_punctuation_00005: test_punctuation_00005</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt8.5')" >
            通过</a>

        <div id='div_pt8.5' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt8.5').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt8.5: case7_5----中英文标点使用混乱-引号错误
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1401, 'keyword': "'", 'suggestion': '', 'tip': '‘’'}
实际返回:{'level': 1, 'type': 1401, 'keyword': "'", 'suggestion': '', 'tip': '‘’'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft8.6' class='none'>
        <td class='failCase'><div class='testcase'>test_punctuation_00006: test_punctuation_00006</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft8.6')" >
            失败</a>

        <div id='div_ft8.6' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft8.6').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft8.6: case7_6----中英文标点使用混乱-书名号错误
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1401, 'keyword': '&lt;', 'suggestion': '《', 'tip': ''}
实际返回:[]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_标点符号.py", line 44, in test_punctuation
    res = response.json()["detections"][0]["corrections"][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_标点符号.py", line 51, in test_punctuation
    self.assertEqual(res, expectResult)
AssertionError: [] != {'level': 1, 'type': 1401, 'keyword': '&lt;', 'suggestion': '《', 'tip': ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt8.7' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_punctuation_00007: test_punctuation_00007</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt8.7')" >
            通过</a>

        <div id='div_pt8.7' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt8.7').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt8.7: case7_7----大数千分位检测-后面不带“元”
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:[]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt8.8' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_punctuation_00008: test_punctuation_00008</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt8.8')" >
            通过</a>

        <div id='div_pt8.8' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt8.8').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt8.8: case7_8----大数千分位检测-后面带“元”
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1201, 'keyword': '12346486', 'suggestion': '12,346,486', 'tip': None}
实际返回:{'level': 1, 'type': 1201, 'keyword': '12346486', 'suggestion': '12,346,486', 'tip': None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft8.9' class='none'>
        <td class='failCase'><div class='testcase'>test_punctuation_00009: test_punctuation_00009</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft8.9')" >
            失败</a>

        <div id='div_ft8.9' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft8.9').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft8.9: case7_9----大数千分位检测-有两个千分符
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:{'level': 1, 'type': 1401, 'keyword': ',', 'suggestion': '，', 'tip': ''}
期望返回:[]
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1401, 'keyword': ',', 'suggestion': '，', 'tip': ''}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_标点符号.py", line 46, in test_punctuation
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1401, 'keyword': ',', 'suggestion': '，', 'tip': ''} != []

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_标点符号.py", line 51, in test_punctuation
    self.assertEqual(res, expectResult)
AssertionError: Lists differ: [{'index': 2, 'corrections': [{'level': 1,[58 chars]'}]}] != []

First list contains 1 additional elements.
First extra element 0:
{'index': 2, 'corrections': [{'level': 1, 'type': 1401, 'keyword': ',', 'suggestion': '，', 'tip': ''}]}

+ []
- [{'corrections': [{'keyword': ',',
-                    'level': 1,
-                    'suggestion': '，',
-                    'tip': '',
-                    'type': 1401}],
-   'index': 2}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft8.10' class='none'>
        <td class='failCase'><div class='testcase'>test_punctuation_00010: test_punctuation_00010</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft8.10')" >
            失败</a>

        <div id='div_ft8.10' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft8.10').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft8.10: case7_10----大数千分位检测-千分符个数不够
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1201, 'keyword': '1560,000', 'suggestion': '1,560,000', 'tip': None}
实际返回:[]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_标点符号.py", line 44, in test_punctuation
    res = response.json()["detections"][0]["corrections"][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_标点符号.py", line 51, in test_punctuation
    self.assertEqual(res, expectResult)
AssertionError: [] != {'level': 1, 'type': 1201, 'keyword': '15[43 chars]None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

<tr class='failClass'>
    <td>Case.case2_文书纠错.test_案号格式.TestLawInfo</td>
    <td>13</td>
    <td>4</td>
    <td>9</td>
    <td>0</td>
    <td><a href="javascript:showClassDetail('c9',13)">详情</a></td>
    <td>&nbsp;</td>
</tr>

    <tr id='ft9.1' class='none'>
        <td class='failCase'><div class='testcase'>test_anhao_00001: test_anhao_00001</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.1')" >
            失败</a>

        <div id='div_ft9.1' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.1').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft9.1: case5_1----中英文标点使用混乱
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1401, 'keyword': '(', 'suggestion': '（', 'tip': ''}
实际返回:{'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None}
期望返回:{'level': 1, 'type': 1401, 'keyword': '(', 'suggestion': '（', 'tip': ''}
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None}, {'level': 1, 'type': 1401, 'keyword': '(', 'suggestion': '（', 'tip': ''}, {'level': 1, 'type': 1401, 'keyword': ')', 'suggestion': '）', 'tip': ''}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 44, in test_anhao
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None} != {'level': 1, 'type': 1401, 'keyword': '(', 'suggestion': '（', 'tip': ''}
- {'keyword': '2021', 'level': 1, 'suggestion': '2022', 'tip': None, 'type': 1002}
?              ^^^^                              ^^^^          ^^^^            ^^

+ {'keyword': '(', 'level': 1, 'suggestion': '（', 'tip': '', 'type': 1401}
?              ^                              ^          ^^           + ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 49, in test_anhao
    self.assertEqual(res, expectResult)
AssertionError: [{'index': 2, 'corrections': [{'level': 1[215 chars]'}]}] != {'level': 1, 'type': 1401, 'keyword': '('[26 chars]: ''}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft9.2' class='none'>
        <td class='failCase'><div class='testcase'>test_anhao_00002: test_anhao_00002</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.2')" >
            失败</a>

        <div id='div_ft9.2' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.2').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft9.2: case5_2----账号与法院代字不匹配
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1003, 'keyword': '最高法', 'suggestion': '闽02', 'tip': None}
实际返回:{'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None}
期望返回:{'level': 1, 'type': 1003, 'keyword': '最高法', 'suggestion': '闽02', 'tip': None}
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None}, {'level': 1, 'type': 1003, 'keyword': '最高法', 'suggestion': '闽02', 'tip': None}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 44, in test_anhao
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None} != {'level': 1, 'type': 1003, 'keyword': '最高法', 'suggestion': '闽02', 'tip': None}
- {'keyword': '2021', 'level': 1, 'suggestion': '2022', 'tip': None, 'type': 1002}
?              ^^^^                              ^ -                            ^

+ {'keyword': '最高法', 'level': 1, 'suggestion': '闽02', 'tip': None, 'type': 1003}
?              ^^^                              ^                             ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 49, in test_anhao
    self.assertEqual(res, expectResult)
AssertionError: [{'index': 2, 'corrections': [{'level': 1[147 chars]e}]}] != {'level': 1, 'type': 1003, 'keyword': '最高[32 chars]None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft9.3' class='none'>
        <td class='failCase'><div class='testcase'>test_anhao_00003: test_anhao_00003</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.3')" >
            失败</a>

        <div id='div_ft9.3' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.3').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft9.3: case5_3----非当前年份
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1002, 'keyword': '2020', 'suggestion': '2021', 'tip': None}
实际返回:{'level': 1, 'type': 1002, 'keyword': '2020', 'suggestion': '2022', 'tip': None}
期望返回:{'level': 1, 'type': 1002, 'keyword': '2020', 'suggestion': '2021', 'tip': None}
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1002, 'keyword': '2020', 'suggestion': '2022', 'tip': None}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 44, in test_anhao
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1002, 'keyword': '2020', 'suggestion': '2022', 'tip': None} != {'level': 1, 'type': 1002, 'keyword': '2020', 'suggestion': '2021', 'tip': None}
- {'keyword': '2020', 'level': 1, 'suggestion': '2022', 'tip': None, 'type': 1002}
?                                                   ^

+ {'keyword': '2020', 'level': 1, 'suggestion': '2021', 'tip': None, 'type': 1002}
?                                                   ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 49, in test_anhao
    self.assertEqual(res, expectResult)
AssertionError: [{'index': 2, 'corrections': [{'level': 1[67 chars]e}]}] != {'level': 1, 'type': 1002, 'keyword': '20[34 chars]None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft9.4' class='none'>
        <td class='failCase'><div class='testcase'>test_anhao_00004: test_anhao_00004</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.4')" >
            失败</a>

        <div id='div_ft9.4' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.4').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft9.4: case5_4----类型代字错误
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1005, 'keyword': '民众', 'suggestion': '', 'tip': None}
实际返回:{'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None}
期望返回:{'level': 1, 'type': 1005, 'keyword': '民众', 'suggestion': '', 'tip': None}
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None}, {'level': 1, 'type': 1005, 'keyword': '民众', 'suggestion': '', 'tip': None}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 44, in test_anhao
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None} != {'level': 1, 'type': 1005, 'keyword': '民众', 'suggestion': '', 'tip': None}
- {'keyword': '2021', 'level': 1, 'suggestion': '2022', 'tip': None, 'type': 1002}
?              ^^^^                              ----                           ^

+ {'keyword': '民众', 'level': 1, 'suggestion': '', 'tip': None, 'type': 1005}
?              ^^                                                         ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 49, in test_anhao
    self.assertEqual(res, expectResult)
AssertionError: [{'index': 2, 'corrections': [{'level': 1[143 chars]e}]}] != {'level': 1, 'type': 1005, 'keyword': '民众[28 chars]None}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft9.5' class='none'>
        <td class='failCase'><div class='testcase'>test_anhao_00005: test_anhao_00005</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.5')" >
            失败</a>

        <div id='div_ft9.5' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.5').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft9.5: case5_5----案号格式正确
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:[]
实际返回:{'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None}
期望返回:[]
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 44, in test_anhao
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None} != []

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 49, in test_anhao
    self.assertEqual(res, expectResult)
AssertionError: Lists differ: [{'index': 2, 'corrections': [{'level': 1,[66 chars]e}]}] != []

First list contains 1 additional elements.
First extra element 0:
{'index': 2, 'corrections': [{'level': 1, 'type': 1002, 'keyword': '2021', 'suggestion': '2022', 'tip': None}]}

+ []
- [{'corrections': [{'keyword': '2021',
-                    'level': 1,
-                    'suggestion': '2022',
-                    'tip': None,
-                    'type': 1002}],
-   'index': 2}]


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt9.6' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_anhao_00006: test_anhao_00006</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt9.6')" >
            通过</a>

        <div id='div_pt9.6' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt9.6').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt9.6: case5_6----顺序错误
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1001, 'keyword': '（2021）民初最高法2071号', 'suggestion': '', 'tip': '正确的案号格式为：（[收案年度]）[法院代字][类型代字][案件编号]号'}
实际返回:{'level': 1, 'type': 1001, 'keyword': '（2021）民初最高法2071号', 'suggestion': '', 'tip': '正确的案号格式为：（[收案年度]）[法院代字][类型代字][案件编号]号'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt9.7' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_anhao_00007: test_anhao_00007</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt9.7')" >
            通过</a>

        <div id='div_pt9.7' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt9.7').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt9.7: case5_7----缺少法院代字
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1001, 'keyword': '（2021）民初2071号', 'suggestion': '', 'tip': '请写明法院'}
实际返回:{'level': 1, 'type': 1001, 'keyword': '（2021）民初2071号', 'suggestion': '', 'tip': '请写明法院'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt9.8' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_anhao_00008: test_anhao_00008</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt9.8')" >
            通过</a>

        <div id='div_pt9.8' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt9.8').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt9.8: case5_8----缺少类型代字
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1001, 'keyword': '（2021）最高法2071号', 'suggestion': '', 'tip': '请写明案件类型'}
实际返回:{'level': 1, 'type': 1001, 'keyword': '（2021）最高法2071号', 'suggestion': '', 'tip': '请写明案件类型'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='pt9.9' class='hiddenRow'>
        <td class='passCase'><div class='testcase'>test_anhao_00009: test_anhao_00009</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt9.9')" >
            通过</a>

        <div id='div_pt9.9' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_pt9.9').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
pt9.9: case5_9----缺少案件编号
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1001, 'keyword': '（2021）最高法民初', 'suggestion': '', 'tip': '请写明案件编号'}
实际返回:{'level': 1, 'type': 1001, 'keyword': '（2021）最高法民初', 'suggestion': '', 'tip': '请写明案件编号'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft9.10' class='none'>
        <td class='failCase'><div class='testcase'>test_anhao_00010: test_anhao_00010</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.10')" >
            失败</a>

        <div id='div_ft9.10' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.10').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft9.10: case5_10----案件编号缺少“号”
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1001, 'keyword': '（2021）最高法民初2071', 'suggestion': '', 'tip': '正确的案号格式为：（[收案年度]）[法院代字][类型代字][案件编号]号'}
实际返回:{'level': 1, 'type': 1001, 'keyword': '（2021）最高法民初2071', 'suggestion': '', 'tip': '请写明案件编号'}
期望返回:{'level': 1, 'type': 1001, 'keyword': '（2021）最高法民初2071', 'suggestion': '', 'tip': '正确的案号格式为：（[收案年度]）[法院代字][类型代字][案件编号]号'}
实际返回:[{'index': 2, 'corrections': [{'level': 1, 'type': 1001, 'keyword': '（2021）最高法民初2071', 'suggestion': '', 'tip': '请写明案件编号'}]}]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 44, in test_anhao
    self.assertEqual(res, expectResult)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.2544.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 837, in assertEqual
    assertion_func(first, second, msg=msg)
AssertionError: {'lev[53 chars]suggestion': '', 'tip': '请写明案件编号'} != {'lev[53 chars]suggestion': '', 'tip': '正确的案号格式为：（[收案年度]）[法院代字][类型代字][案件编号]号'}
  {'keyword': '（2021）最高法民初2071',
   'level': 1,
   'suggestion': '',
-  'tip': '请写明案件编号',
+  'tip': '正确的案号格式为：（[收案年度]）[法院代字][类型代字][案件编号]号',
   'type': 1001}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 49, in test_anhao
    self.assertEqual(res, expectResult)
AssertionError: [{'index': 2, 'corrections': [{'level': 1[79 chars]'}]}] != {'level': 1, 'type': 1001, 'keyword': '（2[75 chars]号]号'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft9.11' class='none'>
        <td class='failCase'><div class='testcase'>test_anhao_00011: test_anhao_00011</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.11')" >
            失败</a>

        <div id='div_ft9.11' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.11').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft9.11: case5_11----缺少收案年度
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1001, 'keyword': '最高法民初2071号', 'suggestion': '', 'tip': '正确的案号格式为：（[收案年度]）[法院代字][类型代字][案件编号]号'}
实际返回:[]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 42, in test_anhao
    res = response.json()["detections"][0]["corrections"][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 49, in test_anhao
    self.assertEqual(res, expectResult)
AssertionError: [] != {'level': 1, 'type': 1001, 'keyword': '最高[70 chars]号]号'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft9.12' class='none'>
        <td class='failCase'><div class='testcase'>test_anhao_00012: test_anhao_00012</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.12')" >
            失败</a>

        <div id='div_ft9.12' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.12').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft9.12: case5_12----收案年度缺少括号
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1001, 'keyword': '2021最高法民初2071号', 'suggestion': '（2021）最高法民初2071号', 'tip': '正确的案号格式为：（[收案年度]）[法院代字][类型代字][案件编号]号'}
实际返回:[]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 42, in test_anhao
    res = response.json()["detections"][0]["corrections"][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 49, in test_anhao
    self.assertEqual(res, expectResult)
AssertionError: [] != {'level': 1, 'type': 1001, 'keyword': '20[90 chars]号]号'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

    <tr id='ft9.13' class='none'>
        <td class='failCase'><div class='testcase'>test_anhao_00013: test_anhao_00013</div></td>
        <td colspan='5' align='center'>

        <!--css div popup start-->
        <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.13')" >
            失败</a>

        <div id='div_ft9.13' class="popup_window">
            <div style='text-align: right; color:red;cursor:pointer'>
            <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.13').style.display = 'none' " >
               [x]</a>
            </div>
            <pre>
            
ft9.13: case5_13----收案年度括号用错
D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\urllib3\connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn((
期望返回:{'level': 1, 'type': 1001, 'keyword': '[2021]最高法民初2071号', 'suggestion': '（2021）最高法民初2071号', 'tip': '正确的案号格式为：（[收案年度]）[法院代字][类型代字][案件编号]号'}
实际返回:[]
Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 42, in test_anhao
    res = response.json()["detections"][0]["corrections"][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\PycharmProjects\unittest\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "D:\Projects\PycharmProjects\unittest\Case\case2_文书纠错\test_案号格式.py", line 49, in test_anhao
    self.assertEqual(res, expectResult)
AssertionError: [] != {'level': 1, 'type': 1001, 'keyword': '[2[92 chars]号]号'}


            </pre>
        </div>
        <!--css div popup end-->
    </td>
    <td></td>
</tr>

<tr id='total_row'>
    <td>总计</td>
    <td>99</td>
    <td>66</td>
    <td>27</td>
    <td>6</td>
	<td>通过率：66.67%</td>
	<td>&nbsp;</td>
</tr>
</table>
<script>
    drawCircle(66, 27, 6)
</script>

<div id='ending'>&nbsp;</div>

</body>
</html>
